src/vorta/config.py

```diff
@@ -1,14 +1,14 @@
 import os
 from pathlib import Path
-import appdirs
+import platformdirs
 
 APP_NAME = 'Vorta'
 APP_AUTHOR = 'BorgBase'
 APP_ID_DARWIN = 'com.borgbase.client.macos'
-dirs = appdirs.AppDirs(APP_NAME, APP_AUTHOR)
-SETTINGS_DIR = dirs.user_data_dir
-LOG_DIR = dirs.user_log_dir
-CACHE_DIR = dirs.user_cache_dir
+dirs = platformdirs.PlatformDirs(APP_NAME, APP_AUTHOR)
+SETTINGS_DIR = dirs.user_data_path
+LOG_DIR = dirs.user_log_path
+CACHE_DIR = dirs.user_cache_path
```

Comment:
Have you checked all uses of these constants for compatibility with this change?

Reply:
I have checked and everything works fine according to me